app/models/destination.rb

```diff
@@ -12,7 +12,8 @@ def metadata
   end
 
   def destinations
-    destinations_list(flow_objects: ordered_flow, current_uuid: flow_uuid)
+    all_flow_objects = ordered_flow + detached
+    destinations_list(flow_objects: all_flow_objects, current_uuid: flow_uuid)
```

Comment:
Do we have tests for the destinations method and detached objects?

Reply:
no we don't. i shall add them